.github/workflows/build-docs.yml

```diff
@@ -56,7 +56,24 @@ jobs:
           cp janssen-*.tgz ../
 
       - name: Generate docs
-        run: echo "Custom work on generating docs can go here."
+        run: |
+         echo "Custom work on generating docs can go here."
+         echo "Generate properties and feature flag documents from elements annotated with @DocFeatureFlag and @DocProperty"
+          
+         # Compile jans-core to pick-up any changes in annotation processors
+         mvn -q -f jans-core/pom.xml -DskipTests clean compile install
+          
+         # Compile modules where classes that use these annotations exist. 
+         # This will generate markdown files under target/classes directory  
+         mvn -q -f jans-auth-server/pom.xml clean compile
+         mvn -q -f jans-fido2/pom.xml clean compile
+         mvn -q -f jans-scim/pom.xml clean compile
+         
+         # Move markdown files to appropriate locations under documentation root 'doc'
+         mv -f jans-auth-server/model/target/classes/janssenauthserver-properties.md docs/admin/reference/json/properties
+         mv -f jans-auth-server/model/target/classes/janssenauthserver-feature-flags.md docs/admin/reference/json/feature-flags
+         mv -f jans-fido2/model/target/classes/fido2-properties.md docs/admin/reference/json/properties
+         mv -f jans-scim/model/target/classes/scim-properties.md docs/admin/reference/json/properties
```

Comment:
please inject a bash script at `https://github.com/JanssenProject/jans/tree/main/automation` under a new folder `docs`. This will allow the workflows to be a little bit cleaner as I suspect we will have more of these flows.

Reply:
@moabu Done. I have moved these commands to a separate script. Please check.